osconfig_tests/test_suites/package_management/package_management_test_data.go

```diff
@@ -40,19 +40,27 @@ func addCreateOsConfigTest(pkgTestSetup []*packageManagementTestSetup) []*packag
 	desc := "test osconfig creation"
 	for _, pkgManager := range pkgManagers {
 		var oc *osconfigpb.OsConfig
+		var image string
 
 		switch pkgManager {
 		case "apt":
+			image = debianImage
 			pkg := osconfigserver.BuildPackage("cowsay")
 			pkgs := []*osconfigpb.Package{pkg}
-			oc = osconfigserver.BuildOsConfig(testName, desc, osconfigserver.BuildAptPackageConfig(pkgs, nil, nil), nil, nil, nil, nil)
+			oc = osconfigserver.BuildOsConfig(fmt.Sprintf("%s-%s", filepath.Base(image), testName), desc, osconfigserver.BuildAptPackageConfig(pkgs, nil, nil), nil, nil, nil, nil)
+			break
```

Comment:
These breaks are pointless, your already at the end of the switch statement.

Reply:
normally any programming language i have used until now are totally opposite of golang syntax. without a break, language like c++/java will fallthrough to the next case, whereas in golang the default behavior is it break and we have special keyword for fallthrough.
still learning golang.
will remove breaks